### PR TITLE
feat(settings): added Recovery key add page

### LINKED ISSUE #4965: [recovery] Create `/account_recovery` page with dummy step 1 of 2 form
The `/account_recovery` route should be hooked up from the settings page in FXA-1614.

Add the text and the form for this page and style it accordingly. This can use the same password input from "change password."

Also allow for the "Cancel" link to take the user back to the settings page.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1654)
┆Issue Number: FXA-1654

### PR DESCRIPTION
**Pull request by dannycoates**
closes #4965

### CODE REVIEW

File: packages/fxa-settings/src/components/PageRecoveryKeyAdd/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import { screen, fireEvent, act } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { MockedCache, renderWithRouter, mockEmail } from '../../models/_mocks';
+import { PageRecoveryKeyAdd } from '.';
+
+// window.console.error = jest.fn();
+
+// afterAll(() => {
+//   (window.console.error as jest.Mock).mockReset();
+// });
```
Comment (chenba) [inline packages/fxa-settings/src/components/PageRecoveryKeyAdd/index.test.tsx:16]: Delete?

**Review by chenba — APPROVED** [review]
LGTM

Still confused whether we should commit yarn.lock changes with feat PRs.

**Comment by jodyheavener** [comment]
I know it's convenient and I might be in the minority, but I'd really like to avoid committing unrelated yarn.lock changes.

**Comment by dannycoates** [comment]
> Still confused whether we should commit yarn.lock changes with feat PRs.

Relax y'all, it's fucking fine

**Pull request merged**